back_end/real_estate_project/settings.py

```diff
@@ -1,12 +1,11 @@
-import os
+from pathlib import Path
 
-# 데이터베이스 설정
-DATABASES = {
-    'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(os.path.dirname(__file__), 'db.sqlite3'),
-    }
-}
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+SECRET_KEY = 'xm3g3j_kzyuk825@52a3e1)pj373w65ufco6o+sd_$vzqg+=2i'
```

Comment:
```suggestion
SECRET_KEY = {$SECRET_KEY}
```
로 변경 부탁드립니다. 시크릿 키 교체 필요합니다.